rules-reviewed/jboss-eap/weblogic/tests/XmlWebLogicRulesTest.windup.test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
I think this file should be renamed to match the naming conventions of the rulesets:

xml-weblogic.windup.test.xml

wdyt?